### PR TITLE
Config

### PR DESCRIPTION
**Pull request by TechStreetDev**

### CODE REVIEW

File: common/src/main/java/io/github/codeutilities/scripts/ScriptHandler.java [hunk 1]
```diff
@@ -39,8 +39,10 @@ public static void init() {
         });
     }
 
-    public static void load() {
+    public static void init() {
```
Comment (Vatten) [inline common/src/main/java/io/github/codeutilities/scripts/ScriptHandler.java]: why?

Comment (TechStreetDev) [reply common/src/main/java/io/github/codeutilities/scripts/ScriptHandler.java]: I did this to match all others in CodeUtilities.java

File: common/src/main/java/io/github/codeutilities/config/impl/MiscellaneousGroup.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import io.github.codeutilities.config.types.IntegerSetting;
+
+public class MiscellaneousGroup extends ConfigGroup {
+    public MiscellaneousGroup(String name) {
+        super(name);
+    }
+
+    @Override
+    public void initialize() {
+        ConfigSubGroup projectaudio = new ConfigSubGroup("projectaudio");
```
Comment (Reasonlesss) [inline common/src/main/java/io/github/codeutilities/config/impl/MiscellaneousGroup.java:17]: project audio?

File: common/src/main/java/io/github/codeutilities/config/internal/settings/OverlayVisibility.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+public enum OverlayVisibility implements IConfigEnum {
+    NONE,
+    ALWAYS,
+    MENU,
+    CHAT;
+
+    @Override
+    public String getKey() {
+        return "projectaudio_overlay_visability";
```
Comment (Reasonlesss) [inline common/src/main/java/io/github/codeutilities/config/internal/settings/OverlayVisibility.java:13]: ????

**Review by Reasonlesss — DISMISSED** [review]

File: common/src/main/java/io/github/codeutilities/config/impl/MiscellaneousGroup.java (hunk 2 above)
Comment (TechStreetDev) [reply common/src/main/java/io/github/codeutilities/config/impl/MiscellaneousGroup.java:17]: its used as a example i had nothing else to register to show an example

**Review by Reasonlesss — APPROVED** [review]

**Review by tk2217 — APPROVED** [review]

**Pull request merged**